src/main/java/org/example/discstorage/UserInfoDiscStorageHandler.java

```diff
@@ -0,0 +1,49 @@
+package org.example.discstorage;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.Set;
+import java.util.TreeSet;
+import org.example.exceptions.InvalidArgument;
+import org.example.model.User;
+
+public class UserInfoDiscStorageHandler {
+  public void saveUserInfoToDiscStorage(Set<User> user) {
+    String filename = "userInfo.ser";
+    try {
+      FileOutputStream file = new FileOutputStream(filename);
+      ObjectOutputStream out = new ObjectOutputStream(file);
+      out.writeObject(user);
+
+      out.close();
+      file.close();
+
+      System.out.println("User Info Saved to the Disk Storage");
+    } catch (FileNotFoundException e) {
+      e.printStackTrace();
```

Comment:
in this case also u have to throw exception as save was not successful

Reply:
wouldn't printing that "Information wasn't save successfully" be good that throwing exception as would terminate the program?

Reply:
catched exception now printing error instead of terminating the whole program